### DISCUSSION TITLE
My gatsby-cli version is decrease after I update it.

### DISCUSSION THREAD

**Discussion by tashasyarifah** [gatsbyjs/gatsby#35971] [Help] [answered]
Hi, I have difficulty updating my gatsby-cli.

At first, my version was 4.5.0. Then, when I update it, for some reason it decreases my version. 

![A screenshot of git bash showed the process of updating gatsby-cli.](https://user-images.githubusercontent.com/66051012/174948347-c9c36300-3ffb-47b8-a2c5-54984d4893e4.png)

I tried to reinstall it yet it still didn't work.

![Another screenshot of gist bash showed the process of reinstalling gatsby-cli.](https://user-images.githubusercontent.com/66051012/174948681-e4314111-6f77-40b1-8fe4-5f368e73c9cb.png)

My OS is windows.

Any help would be helpful.

**Comment by awibox** [comment, marked as answer]
Hi, you should fix a version like `npm i -g gatsby-cli@4.5.0`

**Reply by tashasyarifah** [reply]
Hi @awibox it works! I decided to install the latest version using that command. Thank you!
